readmes/README_ja_JP.md

```diff
@@ -140,7 +140,7 @@ class apache {
 
 このモジュールは[PE対応のすべてのプラットフォーム](https://forge.puppetlabs.com/supported#compat-matrix)上でテスト済みであり、問題は発見されていません。
 
-サポートされているオペレーティングシステムの一覧については、[metadata.json](https://github.com/puppetlabs/puppetlabs-concat/blob/master/metadata.json)を参照してください。
+サポートされているオペレーティングシステムの一覧については、[metadata.json](https://github.com/puppetlabs/puppetlabs-concat/blob//metadata.json)を参照してください。
```

Comment:
URL seems to be missing main. 
blob//metadata.json

Reply:
Fixed